src/OrbitGl/BatcherInterface.h

```diff
@@ -25,15 +24,12 @@ struct PickingUserData {
 
 // Collects primitives to be rendered at a later point in time.
 //
-// By calling BatcherInterface::AddXXX, primitives are added to internal CPU buffers, and sorted
-// into layers formed by equal z-coordinates. Each layer should then be drawn seperately with
-// BatcherInterface::DrawLayer(). BatcherInterface also provides a method to get the user data
-// given a picking_id (in general used for tooltips).
-//
-// BatcherInterface is a pure interface class except for Translations and BatcherId.
+// BatcherInterface is a pure virtual class. By calling BatcherInterface::AddXXX, primitives are
```

Comment:
A class can't be pure virtual. A member function can be pure virtual. The class will be called abstract when it contains at least one pure virtual member function. Hence `pure virtual` -> `abstract` or just say `is an interface class.`